### ISSUE TITLE
Empty "Whitelist(ing) ratio" of nxtool.py

### ISSUE THREAD

**Issue by Loammy** [nbs-system/naxsi#381]
Hi!

I have some whitelist rules and I already put naxsi in enable and disable Learning mode but I can't see the status of whitelist ratio (see below):
![image](https://cloud.githubusercontent.com/assets/29003920/26523237/8c9369fc-42e9-11e7-9a95-81b007e630ab.png)
What I need to do to see this information?

**Comment by buixor** [comment]
Hello,

I didn't understand your question, can you rephrase ?

Regards,

**Comment by Loammy** [comment]
No problem...

I already did the process of creating some whitelist rules and tagged them following documentation found here: https://github.com/nbs-system/naxsi/tree/master/nxapi...
![image](https://cloud.githubusercontent.com/assets/29003920/26678229/dd84f80c-46a5-11e7-8422-3afcf2fdbfa0.png)

When I run "./nxtool.py -c nxapi.json  -x" I can't see the ratio between tagged/untagged events, in my lab always return a empty value, how you can see below:
![image](https://cloud.githubusercontent.com/assets/29003920/26677976/92e2f9bc-46a4-11e7-8188-effbb4da38d8.png)

**Comment by buixor** [comment]
Hello,

Did you called `nxapi` with `--tag` ?
Usually, once you generated your whitelists, you can call

`$ nxtool.py -c ... -w /path/to-whitelists` and you will see output such as :

```
<lot of output>
AVxXvcE0qJG0jumrWUti, AVxXvcrEqJG0jumrWU1R, AVxXvc78qJG0jumrWU4y, AVxXvdAyqJG0jumrWU8H, AVxXvdCsqJG0jumrWU8U, AVxXvdEMqJG0jumrWU8W, AVxXvdEMqJG0jumrWU8Y, AVxXvdIUqJG0jumrWU8k, AVxXvdOzqJG0jumrWU80, Tagged 13161 events out of 32156
```

and if you call it again with `--tag` (in addition to the path to whitelists), it will actually **update** the events in elasticsearch to tag them as **whitelisted**, and this is what the `whitelist(ing) ratio` relies on.

I hope it's more clear like that !

As well, feel free to update the wiki if you don't feel this is clear ;)

**Comment by Loammy** [comment]
Yes, I already have events that are **whitelisted**, in other words, I have events in elasticsearch that are with whitelisted true and others events that are with false. See below:
![image](https://cloud.githubusercontent.com/assets/29003920/26700970/b8f2b952-46f5-11e7-9105-97fba9776201.png)
![image](https://cloud.githubusercontent.com/assets/29003920/26700942/9ee053f8-46f5-11e7-97c0-d02b6b8097e9.png)

In the wiki I saw it
![image](https://cloud.githubusercontent.com/assets/29003920/26701062/16e3d82a-46f6-11e7-876b-60005300de0c.png)
and I don't know why in my lab I don't see this information (relation of events with whitelisted true and false)... My point is only it... I think now I explained better... I hope ;D

**Comment by buixor** [comment]
mh that's pretty weird and I have no idea why.
what version(s) are you using ? (es / naxsi / nxapi etc.)

**Comment by Loammy** [comment]
![image](https://cloud.githubusercontent.com/assets/29003920/26726132/41b9df22-4778-11e7-85ae-cd05c63e054b.png)
![image](https://cloud.githubusercontent.com/assets/29003920/26726177/7f1ec224-4778-11e7-9384-e29beab1b0ee.png)
Naxsi and nxapi I got from this command (is the master version?):
git clone https://github.com/nbs-system/naxsi
![image](https://cloud.githubusercontent.com/assets/29003920/26726852/fb246c6e-477a-11e7-9988-eedf486f8e59.png)

Please, if possible send me the versions that I'll create another lab.

tks

**Comment by buixor** [comment]
mmh I'm using a similar (2.4.1) version here, and I didn't encounter any issues :(

**Comment by Loammy** [comment]
hmmm... Could you please send me your nxtool.py file? Or check line 278?
![image](https://cloud.githubusercontent.com/assets/29003920/26800151/97e7e0e0-4a0e-11e7-94f1-9f1ffdebf2fb.png)
In the line 278 is the only reference of whitelisted that I found in the nxtool.py file...

**Comment by jreisinger** [comment]
I am having the same issue. Elasticsearch: 5.4.3, nginx: 1.11.8.

**Comment by buixor** [comment]
fixed by merrge
